client/src/App.js

```diff
@@ -1,13 +1,37 @@
-import React from 'react';
+import React, { Component } from 'react';
```

Comment:
Some files still need to be changed to match up with master, but glad you could get most of it fixed

Reply:
pulled new files, so i think i should be up to date now